src/hub/tests/conftest.py

```diff
@@ -37,32 +45,61 @@ def pytest_configure():
     os.environ["ALLOWED_ORIGIN_SYSTEMS"] = "Test_system,Test_System,Test_System1"
     sys._called_from_test = True
 
-    # Locate absolute path of dynalite
-    dynalite = f"{CFG.REPO_ROOT}/node_modules/.bin/dynalite"
 
-    cmd = f"{dynalite} --port {CFG.DYNALITE_PORT}"
-    ddb_process = subprocess.Popen(
-        cmd, shell=True, env=os.environ, stdout=subprocess.PIPE
+def get_free_tcp_port():
+    import socket
+
+    tcp = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+    tcp.bind(("", 0))
+    addr, port = tcp.getsockname()
+    tcp.close()
+    return port
+
+
+def _docker_client():
+    return docker.from_env()
```

Comment:
what is the value of this function?

Reply:
https://sourcegraph.com/github.com/docker/docker-py/-/blob/docker/client.py#L43

Reply:
? i dont know what the code reference was supposed to mean to me; why have a private (starting with an '_') function to return the result of a function off of what appears to be a global `docker` object?  Why not just just the body of this function, inline at the call sites for _docker_client() ?